device/mellanox/x86_64-mlnx_msn4600-r0/ACS-MSN4600/hwsku.json

```diff
@@ -0,0 +1,196 @@
+        }, 
+        "Ethernet296": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet304": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet312": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet320": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet328": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet336": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet344": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet352": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet360": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet368": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet376": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet384": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet392": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet400": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet408": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet416": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet424": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet432": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet440": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet448": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet456": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet464": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet472": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet480": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet488": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet496": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }, 
+        "Ethernet504": {
+            "default_brkout_mode": "1x200G[100G,50G,40G,25G,10G,1G]"
+        }
+    }
+}
```

Comment:
This file is for the dynamic break out. meaning that we need to remove it from the basic PR so it can be merged to 202012 and have the DPB support on a seperated PR.